.changeset/moody-plants-add.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Increased and standardized gap in button groups
```

Comment:
```suggestion
Increased the gap between primary and secondary buttons in `Hero`, `CTABanner` and `ButtonGroup` components from `8px` to `16px`.
```

Reply:
For clarity in the changelog entry and making it easier for release / integration conductor to know what to test ☝️